src/obfl-specification.html

```diff
@@ -1209,7 +1211,7 @@ <h4 id="L1207" class="include">leader</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item</dd>
+    <dd>before, after, toc-entry, block, toc-block, item</dd>
```

Comment:
If I understood this definition correctly this is not correct.

```leader``` may not be used directly inside of a toc-block

Reply:
Agree with Daniel.

Reply:
I made changes so that `toc-block` has only these child elements: `toc-entry`, `toc-block`